whiskey.yml

```diff
@@ -19,7 +19,14 @@ Build:
       Argument: [ UpdateBuild, -Version, "$(WHISKEY_SEMVER2)+$(WHISKEY_BUILD_NUMBER)" ]
 
 - PowerShell:
-    ScriptBlock: prism install
+    ScriptBlock: |
+        if( -not (Get-Command -Name 'prism' -ErrorAction Ignore) )
+        {
+            $module = Find-Module -Name 'Prism' -MinimumVersion '0.4.0' | Select-Object -First 1
+            Set-PSRepository -Name $module.Repository -InstallationPolicy Trusted
+            $module | Install-Module -AllowClobber -Scope CurrentUser
+        }
+        prism install | Format-Table -Auto

```

Comment:
I thought prism replaced with PowerShellGet? Or what it the other way around..

Reply:
No prism uses PowerShellGet under to hood to install things.